research/fed-bpt/src/global_es.py

```diff
@@ -95,7 +95,7 @@ def run(self) -> None:
 
             clients = self.sample_clients(self.num_clients)
 
-            global_model = FLModel(params={"global_es": global_es})
+            global_model = FLModel(params={"global_es": global_es}, current_round=self.current_round)
```

Comment:
Is there any other places we need to change?

@holgerroth @SYangster @ZiyueXu77

Reply:
Now it is important the FLModel object MUST have the current round info!

Reply:
Right, passing in the round will be needed for all workflows that want to use the IntimeModelSelector for model selection

Reply:
I have searched the usage. No other places need to add this current_round data when creating the FLModel object.